src/transports/ipc.rs

```diff
@@ -60,22 +60,24 @@ impl Drop for EventLoopHandle {
 }
 
 
-/// Error returned while initializing IPC transport
+/// Error returned while initializing IPC transport.
 pub type IpcError = io::Error;
+/// Result of initializing IPC transport.
+pub type IpcResult<T> = ::std::result::Result<T, IpcError>;
```

Comment:
Style is typically to declare `type Result` and then it can be used as `ipc::Result`